test-specifications/token-service/tokenCreateTransaction.md

```diff
@@ -0,0 +1,1150 @@
+|----------------|--------|-------------------------------------------------------------------------------------|
+| tokenId        | string | The ID of the created token.                                                        |
+| status         | string | The status of the submitted `TokenCreateTransaction` (from a `TransactionReceipt`). |
+
+## Property Tests
+
+### **Name:**
+
+- The name for the new token.
+
+| Test no | Name                                                        | Input                                                                                                                                                                    | Expected response                                                                                                                                               | Implemented (Y/N) |
+|---------|-------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| 1       | Creates a token with a name that is a valid length          | name="testname", symbol="testsymbol", treasuryAccount=<OPERATOR_ACCOUNT_ID>                                                                                              | The token creation succeeds and the token's name equals "testname".                                                                                             | N                 |
+| 2       | Creates a token with a name that is the minimum length      | name="t", symbol="testsymbol", treasuryAccount=<OPERATOR_ACCOUNT_ID>                                                                                                     | The token creation succeeds and the token's name equals "t".                                                                                                    | N                 |
+| 3       | Creates a token with a name that is empty                   | name="", symbol="testsymbol", treasuryAccount=<OPERATOR_ACCOUNT_ID>                                                                                                      | The token creation fails with a MISSING_TOKEN_NAME response code from the network.                                                                              | N                 |
+| 4       | Creates a token with a name that is the maximum length      | name="This is a really long name but it is still valid because it is 100 characters exactly on the money!!", symbol="testsymbol", treasuryAccount=<OPERATOR_ACCOUNT_ID>  | The token creation succeeds and the token's name equals "This is a really long name but it is still valid because it is 100 characters exactly on the money!!". | N                 |
```

Comment:
The length limit is not 100 characters, it's 100 _bytes_ "when the string is encoded UTF-8".
Testing that, however, is harder.  For tests where all of the characters are in the LATIN-1 group, 100 _characters_ should be OK.
To test more thoroughly, include some Linear-b Ideograms in the text (typically those are 4 bytes each in UTF-8).

Reply:
You're right, this should be tested. I think it may be better to make a separate issue to add those tests as I'd want to add them to some other transactions as well that are out of the scope of this PR